miso-web/src/it/java/uk/ac/bbsrc/tgac/miso/webapp/integrationtest/page/AbstractListPage.java

```diff
@@ -101,6 +101,22 @@
     public static final String INDICES = "indices";
   }
 
+  public static class ButtonText {
+    public static final String CREATE = "Create";
+    public static final String EDIT = "Edit";
+    public static final String PROPAGATE = "Propagate";
+    public static final String PRINT_BARCODES = "Print Barcode(s)";
+    public static final String ADD_QCS = "Add QCs";
+    public static final String EDIT_QCS = "Edit QCs";
+    public static final String RECEIVE = "Receive";
+    public static final String MAKE_DILUTIONS = "Make dilutions";
+    public static final String POOL_TOGETHER = "Pool together";
+    public static final String POOL_SEPARATELY = "Pool separately";
```

Comment:
`ButtonText` should be on specific pages since most of them don't make sense for most of the lists

Reply:
We don't instantiate more specific pages (there is no ListSamplesPage) so I'm going to leave them here.